examples/integration/test/fixtures/bwb.xcodeproj/xcshareddata/xcschemes/ABundle.xcscheme

```diff
@@ -14,9 +14,9 @@
                <EnvironmentBuildable>
                   <BuildableReference
                      BuildableIdentifier = "primary"
-                     BlueprintIdentifier = "C9EB85EDD4934E0A73AC195A"
+                     BlueprintIdentifier = "A8A13605CC563E9B6464CD8D"
                      BuildableName = "ABundle.aplugin"
-                     BlueprintName = "Bundle"
+                     BlueprintName = "ABundle"
```

Comment:
Example 1: `ABundle.aplugin` is the product, so it makes sense that this target is `ABundle` instead of `Bundle`.